jina/optimizers/parameters.py

```diff
@@ -35,16 +35,6 @@ def __init__(
         # The step != 1 and log arguments cannot be used at the same time.
         # To set the log argument to True, set the step argument to 1.
         self.log = log
-        self.optuna_method = 'suggest_int'
-
-    def to_optuna_args(self):
-        return {
-            'name': self.jaml_variable,
-            'low': self.low,
-            'high': self.high,
-            'step': self.step_size,
-            'log': self.log,
-        }
 
 
 class FloatParameter(OptimizationParameter):
```

Comment:
just to know, we are not going to do the separation of "parameters" and "sampling" before 1.0 right?

Reply:
Was not planned. There is a reason, optuna keeps them together: Even if we separate them, they will still be glued together, since there is no point having a `Uniform` sampler for categorical parameters. There it would be a `DiscreteUniform` one, which optuna exactly does.